src/main/java/org/opentripplanner/netex/loader/mapping/StopTimesMapper.java

```diff
@@ -76,7 +87,13 @@ MappedStopTimes mapToStopTimes(
             String pointInJourneyPattern = currentPassingTime.getPointInJourneyPatternRef().getValue().getRef();
 
             StopPointInJourneyPattern stopPoint = findStopPoint(pointInJourneyPattern, journeyPattern);
-            Stop stop = lookupStop(stopPoint, quayIdByStopPointRef, stopsById);
+            StopLocation stop = lookUpStopLocation(
+                stopPoint,
+                quayIdByStopPointRef,
+                flexibleStopPlaceIdByStopPointRef,
+                stopsById,
+                flexibleStopLocationsById
```

Comment:
Class member fields passed to a class member function. No need to pass these on.